src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-document-settings-modal/sw-order-document-settings-modal.spec.js

```diff
@@ -213,4 +212,81 @@ describe('src/module/sw-order/component/sw-order-document-settings-modal', () =>
         await documentNumberFieldInput.setValue('Prefix-1000-Suffix');
         expect(documentNumberFieldInput.element.value).toBe('Prefix-1000-Suffix');
     });
+
+    it('should enable & disable by documentNumber value', async () => {
```

Comment:
```suggestion
    it('should enable & disable create & preview buttons by documentNumber value', async () => {
```
like in the other tests, it becomes clearer at first glance